samples/parallel-processing-california-housing-data/README.md

```diff
@@ -0,0 +1,140 @@
+    ```
+
+## Running locally
+
+1. Make a copy of [.env.example](./local_development/.env.example), place it in the root of this sample, configure the variables, and rename the file to .env.
+
+1. Update variable values. 
+
+    | name | description |
+    |---|---|
+    |SUBSCRIPTION_ID|Azure Subscription ID|
+    |RESOURCE_GROUP| Azure Resource group name |
+    |WORKSPACE_NAME| Azure Machine Learning workspace name|
+    |AML_ENV_NAME| Azure Machine Learning Environment name|
+    |AML_COMPUTE_CLUSTER_NAME| Azure Machine Learning compute cluster name|
+    |AML_COMPUTE_VM_SIZE|Azure Machine Learning compute virtual machine size. i.e. "STANDARD_D2_V2"|
+    |AML_COMPUTE_IDLE_TIME|Azure Machine Learning compute idle time (seconds) before scaling down|
+    |AML_COMPUTE_MIN_NODES|Azure Machine Learning compute minimum nodes number|
+    |AML_COMPUTE_MAX_NODES|Azure Machine Learning compute maximum nodes number|
+    |AML_BLOB_DATASTORE_NAME| Azure Machine Learning blob datastore name|
+    |AML_STORAGE_ACCOUNT_NAME| Azure Storage Account name for Azure Machine Learning blob datastore|
+    |AML_BLOB_CONTAINER_NAME | Blob container name which contains input data|
+    |AML_STORAGE_ACCOUNT_KEY |Azure Storage Account Key|
+    |PIPELINE_ENDPOINT_NAME| Azure Machine Learning pipeline endpoint name|
+    |PIPELINE_NAME| Azure Machine Learning pipeline name|
+    |INPUT_DIR| folder name which is used to save input files. Don't change this value.|
+    |SOURCES_DIR_TRAIN|source code directory for Azure Machine Learning pipeline|
+    |PREPARATION_STEP_SCRIPT_PATH| python script path for the preparation step|
+    |EXTRACTION_STEP_SCRIPT_PATH| python script path for the extraction step|
+    |TRAINING_STEP_SCRIPT_PATH| python script path for the training step|
+    |ERROR_THRESHOLD|The number of record failures for TabularDataset and file failures for FileDataset that should be ignored during processing. If the error count goes above this value, then the job will be aborted.|
+    |NODE_COUNT|Number of nodes in the compute target used for running the Parallel Run|
+    |MINI_BATCH_SIZE|size of data can be processed in one run() call|
+    |PROCESS_COUNT_PER_NODE|Number of processes executed on each node. Optional, default value is number of cores on node|
+    |RUN_INVOCATION_TIMEOUT|Timeout in seconds for each invocation of the run() method|
+    
+
+1. Use the VSCode [dev container](./.devcontainer), or install Anaconda or Mini Conda and create a Conda envrionment by running [local_install_requirements.sh](./local_development/local_install_requirements.sh).
+
+1. In VSCode, open the root folder of this sample, select the Conda environment created above as the Python interpretor.
+
+1. Publish and run Azure ML pipelines. 
+    - To publish and run Azure ML pipelines, run:
+        ```shell
+        # publish the Azure ML pipeline
+        python -m ml_service.pipelines.build_pipeline
+        ```
+
```

Comment:
Would be really great to have a section here explaining briefly the additional parameters for the ParallelScriptStep. I made a comment on the `.env.example` as well.

Reply:
updated in the new commit https://github.com/liupeirong/MLOpsManufacturing/pull/69/commits/5a6dbbf4d8bc97a67a37db14f1a99dc50637c5e2. I have separated the parallel run variable explanation from other variables. and also added more details for each of them.